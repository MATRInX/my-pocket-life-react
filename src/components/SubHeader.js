import React from 'react';
import SubHeaderLogo from './SubHeaderLogo';
import SubHeaderNavBar from './SubHeaderNavBar';

const SubHeader = () => (
  <div>
    <SubHeaderLogo />
    <SubHeaderNavBar />
  </div>
);

export default SubHeader;