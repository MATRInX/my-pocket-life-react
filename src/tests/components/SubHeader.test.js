import React from 'react';
import { shallow } from 'enzyme';
import SubHeader from '../../components/SubHeader';

test('should correctly render SubHeader', () => {
  const wrapper = shallow(<SubHeader />);
  expect(wrapper).toMatchSnapshot();
});