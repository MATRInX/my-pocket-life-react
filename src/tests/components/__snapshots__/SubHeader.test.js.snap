// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render SubHeader 1`] = `
<div>
  <SubHeaderLogo />
  <SubHeaderNavBar />
</div>
`;
